app/views/topics/show.html.erb

```diff
@@ -8,6 +8,7 @@
     <% end  %>
   </div>
   <div class="col-md-4">
-    <h1>link to delete</h1>
+    <%= link_to "Edit Topic", edit_topic_path, class: "btn btn-block btn-lg btn-warning" %>
```

Comment:
It does generate the expected path. This link is at
http://localhost:3000/topics/1. And the edit url is
http://localhost:3000/topics/1/edit

On Fri, Feb 27, 2015 at 8:31 AM, Eliot Sykes <notifications@github.com>
wrote:

> In app/views/topics/show.html.erb
> <https://github.com/LuisMenjivar/luisbookmark/pull/6#discussion_r25505829>
> :
>
> > @@ -8,6 +8,7 @@
> >      <% end  %>
> >    </div>
> >    <div class="col-md-4">
> > -    <h1>link to delete</h1>
> > +    <%= link_to "Edit Topic", edit_topic_path, class: "btn btn-block btn-lg btn-warning" %>
>
> Does edit_topic_path generate the expected path? It looks like this is
> missing information.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/luisbookmark/pull/6/files#r25505829>.
>